sdcm/utils/csrangehistogram.py

```diff
@@ -229,25 +229,62 @@ def build_from_log_line(self, log_line: str, hst_log_start_time: float) -> dict[
 
         return _CSRangeHistogramBuilder._get_summary_for_operation_by_hdr_tag(histogram)
 
-    def _build_histogram_from_file(self, hdr_file: str, hdr_tag: str) -> _CSRangeHistogram:
+    @staticmethod
+    def switch_tag(hdr_tag):
+        tag_pairs = [[_CSHistogramThroughputTags.WRITE.value, CSHistogramTags.WRITE.value],
+                     [_CSHistogramThroughputTags.READ.value, CSHistogramTags.READ.value]]
+        for pair in tag_pairs:
+            if hdr_tag in pair:
+                return [tag for tag in pair if tag != hdr_tag][0]
+        return None
```

Comment:
why not just a mapping like that:
```python
def switch_tag(hdr_tag):
    mapping = {_CSHistogramThroughputTags.WRITE.value: CSHistogramTags.WRITE.value,
                       _CSHistogramThroughputTags.READ.value: CSHistogramTags.READ.value}
    # extend it with the reverse mapping
    mapping.update(dict(map(reversed, mapping.items())))
    return mapping.get(hdr_tag)
```

Reply:
also it's might be called `fallback_tag` (if at some point we'll have more then 2 available)

Reply:
do you mean it may be something except of throughput and latency?

Reply:
> why not just a mapping like that:
> 
> ```python
> def switch_tag(hdr_tag):
>     mapping = {_CSHistogramThroughputTags.WRITE.value: CSHistogramTags.WRITE.value,
>                        _CSHistogramThroughputTags.READ.value: CSHistogramTags.READ.value}
>     # extend it with the reverse mapping
>     mapping.update(dict(map(reversed, mapping.items())))
>     return mapping.get(hdr_tag)
> ```

done

Reply:
yes, right now there are 3 types of tags
there one for the wait time, but we don't care about it on it's own.

tomorrow we'll send add some other metric for some reason into the hdr (again maybe not)

Reply:
@fruch 
I changed the function. Please, review